### PR TITLE
[DO - 1383] Create Daily payments report logic

### PR DESCRIPTION
**Pull request by Andrii17s**

### CODE REVIEW

File: payment_system/emails.py [hunk 1]
```diff
@@ -260,3 +261,14 @@ def new_custom_sub_request(custom_subscription_request: 'CustomSubscriptionReque
             'csr': custom_subscription_request,
         },
     )
+
+
+def create_report(daily_report: 'InvoiceDailyReport'):
```
Comment (roman-tiukh) [inline payment_system/emails.py:266]: you declare type `InvoiceDailyReport` but pass string on call
```
emails.create_report(message_text)
```

File: payment_system/emails.py [hunk 2]
```diff
@@ -260,3 +261,14 @@ def new_custom_sub_request(custom_subscription_request: 'CustomSubscriptionReque
...
+    send_template_mail(
+        to=[settings.SUPPORT_EMAIL],
+        subject='Підсумок оплати інвойсів за ' + f'{timezone.localdate()}',
+        template='payment_system/emails/new_daily_report.html',
```
Comment (roman-tiukh) [inline payment_system/emails.py:270]:
```suggestion
        template='payment_system/emails/daily_report.html',
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
Attach a screenshot to the PR.
What the report looks like

**Comment by Andrii17s** [comment]
Screenshot from 2021-04-17 17-57-58

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Comment by Andrii17s** [comment]
Screenshot from 2021-04-12 16-18-30

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**